### PR TITLE
Fix Net loading

### PR DESCRIPTION
**Pull request by simanw304**
resolve this issue: https://github.com/intel-analytics/zoo/issues/277
and https://github.com/intel-analytics/zoo/issues/276

### CODE REVIEW

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/NetSpec.scala [hunk 1]
```diff
@@ -93,4 +93,24 @@ class NetSpec extends ZooSpecHelper{
     val newModel = model.newGraph("LeNet/fc3/Relu")
     newModel.outputNodes.head.element.getName() should be ("LeNet/fc3/Relu")
   }
+
+  "net load model" should "work properly" in {
+    val seq = Sequential[Float]().add(Dense[Float](3, inputShape = Shape(2, 3)))
+
+    val tmpFile1 = createTmpFile()
+    val absPath1 = tmpFile1.getAbsolutePath
+    seq.saveModule(absPath1, overWrite = true)
```
Comment (zhichao-li) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/NetSpec.scala]: Remove the code of saving model

**Comment by hkvision** [comment]
But the issue is only calling load will have an error.
Adding unit tests only won't solve the problem.

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/NetSpec.scala [hunk 2]
```diff
@@ -93,4 +93,14 @@ class NetSpec extends ZooSpecHelper{
     val newModel = model.newGraph("LeNet/fc3/Relu")
     newModel.outputNodes.head.element.getName() should be ("LeNet/fc3/Relu")
   }
+
+  "net load model" should "work properly" in {
+    val resource = getClass().getClassLoader().getResource("models")
+    val path = resource.getPath + "/" + "zoo_keras"
+
+    val seq = Sequential[Float]().add(Dense[Float](3, inputShape = Shape(2, 3)))
+
```
Comment (zhichao-li) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/NetSpec.scala]: add seq.forward

File: zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/NetSpec.scala [hunk 3]
```diff
@@ -93,4 +93,15 @@ class NetSpec extends ZooSpecHelper{
     val newModel = model.newGraph("LeNet/fc3/Relu")
     newModel.outputNodes.head.element.getName() should be ("LeNet/fc3/Relu")
   }
+
+  "net load model" should "work properly" in {
+    val resource = getClass().getClassLoader().getResource("models")
+    val path = resource.getPath + "/" + "zoo_keras"
+
+    val seq = Sequential[Float]().add(Dense[Float](3, inputShape = Shape(2, 3)))
```
Comment (zhichao-li) [inline zoo/src/test/scala/com/intel/analytics/zoo/pipeline/api/NetSpec.scala]: remove this line

**Comment by hkvision** [comment]
Update here as well https://github.com/intel-analytics/analytics-zoo/blob/master/zoo/src/main/scala/com/intel/analytics/zoo/models/common/ZooModel.scala#L93 
Thanks!

**Comment by simanw304** [comment]
jenkins passed http://172.168.2.101:8080/view/ZOO-PR-PIPELINE/523/

**Pull request merged**